sql/plan/ddl.go

```diff
@@ -0,0 +1,59 @@
+package plan
+
+import (
+	"fmt"
+
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+// CreateTable is a node describing the creation of some table.
+type CreateTable struct {
+	Database sql.Database
+	name     string
+	schema   sql.Schema
+}
+
+// NewCreateTable creates a new CreateTable node
+func NewCreateTable(db sql.Database, name string, schema sql.Schema) *CreateTable {
+	return &CreateTable{
+		Database: db,
+		name:     name,
+		schema:   schema,
```

Comment:
I'm not sure it's needed but here we can set `Source` for each column with the name of the table.